piksi_tools/serial_link.py

```diff
@@ -381,21 +357,19 @@ def main(args):
     with Handler(Framer(driver.read, driver.write, args.verbose)) as link:
         # Logger with context
         with get_logger(args.log, log_filename, args.expand_json) as logger:
-            with get_append_logger(append_log_filename, tags) as append_logger:
-                link.add_callback(printer, SBP_MSG_PRINT_DEP)
-                link.add_callback(log_printer, SBP_MSG_LOG)
-                Forwarder(link, logger).start()
-                Forwarder(link, append_logger).start()
-                if use_broker and base and serial_id:
-                    device_id = get_uuid(channel, serial_id)
-                    with HTTPDriver(str(device_id), base) as http:
-                        with Handler(
-                                Framer(http.read, http.write,
-                                       args.verbose)) as slink:
-                            Forwarder(slink, swriter(link)).start()
-                            run(args, link)
-                else:
-                    run(args, link)
+            link.add_callback(printer, SBP_MSG_PRINT_DEP)
+            link.add_callback(log_printer, SBP_MSG_LOG)
+            Forwarder(link, logger).start()
```

Comment:
I think we can also remove this HTTP Forwarder feature longterm as well.  Doesn't need to be done as part of this effort but it isn't in use to my knowledge and causes us to import requests just to run serial_link.py even if it isn't in use.

Reply:
This would allow us to remove a few more command line options to serial link and harmonize serial link's initialization with the other tools

Reply:
good pointers. I created a follow-up task to do these at some point in the future, after the python 3 update: https://swift-nav.atlassian.net/browse/ESD-1035